charts/nextcloud/templates/deployment.yaml

```diff
@@ -325,6 +347,20 @@ spec:
       - name: nextcloud-nginx-config
         configMap:
           name: {{ template "nextcloud.fullname" . }}-nginxconfig
+          defaultMode: 420
+          items:
+            - key: nginx.conf
+              mode: 292
+              path: nginx.conf
```

Comment:
Was this broken before?

Reply:
Because I added the metrics configuration to the existing configuration file, I need to specify both via items instead of just mounting the entire ConfigMap. Otherwise metrics will be mounted as well, even though it may not have been enabled.
So not broken, but now necessary to specify which keys to mount.